train.py

```diff
@@ -233,14 +239,41 @@ def train(hyp, opt, device, tb_writer=None):
     model.class_weights = labels_to_class_weights(dataset.labels, nc).to(device) * nc  # attach class weights
     model.names = names
 
+    qat = False
```

Comment:
nit: move this variable to right before you set it at the bottom of this block at 
```if manager.quantization_modifiers:```